app/src/main/java/is/xyz/filepicker/FilePickerFragment.java

```diff
@@ -182,12 +183,12 @@ public String getFullPath(@NonNull final File path) {
     /**
      * Get the root path.
      *
-     * @return the highest allowed path, which is "/" by default
+     * @return the highest allowed path, which is external storage by default
      */
     @NonNull
     @Override
     public File getRoot() {
-        return new File("/");
+        return new File(Environment.getExternalStorageDirectory().getPath());
```

Comment:
Given that it's a "root" folder wouldn't that make it impossible to navigate to e.g. external sd card?

Reply:
External storage is precisely where navigation begins with this change. I personally do not own any phone that takes a microSD card so I cannot quite ensure how transitioning between those and internal storage will work (if at all) but I find this current compromise quite favourable as opposed to having users navigate through their entire UNIX directory structure of inaccessible files to find their media.

Reply:
An external microsd card isn't external storage in Android terms anymore (thanks google). The problem I see is that you need to go up a directory to get to the microsd, which this would forbid. In fact, the behavior should already be to start navigation in the external storage directory as per https://github.com/mpv-android/mpv-android/blob/db2ecf5918d9795c8f54bbef0a16b93eaa39d2a8/app/src/main/java/is/xyz/mpv/MainActivity.kt#L32 but perhaps it's bugged.

Reply:
> I personally do not own any phone that takes a microSD card

I do and being unable to play any files on there (I assume) is not a <i>compromise</i> at all.

> having users navigate through their entire UNIX directory structure of inaccessible files to find their media.

The file browser starts at `/storage/emulated/0` for me, which is the internal storage.

Reply:
Yes, I just tested all my devices again and appears that the bug I fixed months ago was likely just a transient problem with my phone since I can't reproduce it anymore. Will drop this patch.